react/src/base/inputs/components/SprkInputElement/SprkInputElement.js

```diff
@@ -73,9 +86,9 @@ class SprkInputElement extends Component {
         data-id={idString}
         data-analytics={analyticsString}
         aria-invalid={!valid}
-        aria-describedby={errorContainerId}
+        aria-describedby={getAriaDescribedBy()}
```

Comment:
Thanks for adding this. I did notice a potential issue. This seems to be running every time the component renders for me, so even though the errorcontainer /id hasnt changed this gets triggered. Every time I type a character in the input, it runs. Which seems to drag down performance. Maybe I am doing something weird? But thats what I am seeing.  I think something to try here would be to bind this to a method that runs on one of the lifecycle hooks, componentDidMount() ?